examples/hybrid_quantum_algorithms/QAOA/utils_classical.py

```diff
@@ -6,8 +6,6 @@
 import networkx as nx
 from pyqubo import Spin, solve_ising
 
-warnings.filterwarnings("ignore", category=matplotlib.cbook.mplDeprecation)
```

Comment:
Why did you remove this?

Reply:
This is no longer needed: https://matplotlib.org/stable/api/prev_api_changes/api_changes_3.7.0.html#deprecation-aliases-in-cbook